format/FlightSql.proto

```diff
@@ -441,7 +441,7 @@ enum SqlInfo {
    * - return 3   (\b11)    => [SQL_SUBQUERIES_IN_COMPARISONS, SQL_SUBQUERIES_IN_EXISTS];
    * - return 4   (\b100)   => [SQL_SUBQUERIES_IN_INS];
    * - return 5   (\b101)   => [SQL_SUBQUERIES_IN_COMPARISONS, SQL_SUBQUERIES_IN_INS];
-   * - return 6   (\b110)   => [SQL_SUBQUERIES_IN_INS, SQL_SUBQUERIES_IN_EXISTS];
+   * - return 6   (\b110)   => [SQL_SUBQUERIES_IN_COMPARISONS, SQL_SUBQUERIES_IN_EXISTS];
```

Comment:
Is this change valid? Judging by the above, SQL_SUBQUERIES_IN_COMPARISONS is 0b001 while SQL_SUBQUERIES_IN_INS is 0b100.

Reply:
Yes, this seems wrong. Additionally the enum values are sequential (0, 1, 2, 3) not bitmasks (unless those values are intended to be bit indices in which case we should document them as such).

Reply:
@jcralmeida any comment here? I forgot about this, let's fix these last few things and then kick off the vote

Reply:
Oh sorry @lidavidm , I've changed back to how it was which was the correct one. I don't know how/why I've changed it